redirects-list.json

```diff
@@ -43,11 +43,11 @@
   "^/doc/start/data-versioning$                                                           /doc/start/data-management",
   "^/doc/start/data-and-model-versioning(/.*)?$                                           /doc/start/data-management 302",
   "^/doc/start/sharing-data-and-model-files$                                              /doc/start/data-management#storing-and-sharing 302",
-  "^/doc/start/data-access$                                                               /doc/start/data-management/data-and-model-access",
-  "^/doc/start/data-and-model-access(/.*)?$                                               /doc/start/data-management/data-and-model-access 302",
+  "^/doc/start/data-access$                                                               /doc/user-guide/data-management/discovering-and-accessing-data",
+  "^/doc/start/data-and-model-access(/.*)?$                                               /doc/user-guide/data-management/discovering-and-accessing-data 302",
   "^/doc/start/data-pipelines(/.*)?$                                                      /doc/start/data-management/data-pipelines 302",
   "^/doc/start/metrics-parameters-plots(/.*)?$                                            /doc/start/data-management/metrics-parameters-plots 302",
-  "^/doc/start/experiment-management(/.*)?$                                               /doc/start/experiments",
```

Comment:
if you are updating 301s I think you need to an extra one  /doc/start/experiments -> to something new. Overall, we should be using 302 in most cases where we iterate on the target and expect target to keep changing.

Reply:
This line change is a mistake I forgot to undo! reverted. I'm not doing anything fancy here, no need to add a redirect, `/doc/start/experiments` is the same before and after this PR (I changed to experiment-management and then back because I saw that it is confusing because of a user-guide page with the same name)